2.6.0/spec/services.md

```diff
@@ -0,0 +1,269 @@
+# Services Endpoint
+
+Used to fetch a list of all available services from a CO belonging to the current SP.
+
+## Methods
+
+### GET
+
+#### Available services for a single Access
+
+To get available services for a single access use accessId as key. The response contains details about activated services and the services
+ available for activation.
+ 
+Request
+```http
+GET /onapi/2.6/services/{accessId} HTTP/1.1
+```
+
+Response 
+```HTTP
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+#### Example data from a single access response 
+
+When a single access is requested the response contains a single JSON-object containing the accessId and an array of services.
+
+```JSON
+{
+  "accessId": "8732c2f065e2490babce820e94b1011a",
+  "services": [
```

Comment:
I have a weak preference for just having a list of services with the `accessId` inside of each object.

Reply:
The base for this implementation was to make is easy to migrate from the current accesses format where accessId and the services array already is as above. Get an access, and then store the services in your local db for cache. If the accessId is embedded it might be more inefficiant as you might need to process the entire file in order to get a complate services list to store.

Reply:
That makes sense. I'm used to relational databases where each service on each access would just be another row to process, no need to do it in any particular order. But if you have a hierarchical database structure (or just store the available services as a json blob on each access) that might not be as easy.